tests/manager/bug0940-001.phpt

```diff
@@ -6,13 +6,12 @@ PHPC-940: php_phongo_free_ssl_opt() attempts to free interned strings
 --FILE--
 <?php
 
-var_dump(new MongoDB\Driver\Manager(null, [], ['ca_file' => false]));
+var_dump(new MongoDB\Driver\Manager(null, ['tlsCAFile' => false]));
```

Comment:
This causes a test failure as a string is expected.

Reply:
This required a more involved fix, as I had to preserve use of a driver option to ensure that `php_phongo_make_ssl_opt()` actually does something. The original patch for [PHPC-940](https://jira.mongodb.org/browse/PHPC-940) may no longer apply since `php_phongo_fetch_string()` always returns an allocated string, but there's no harm in continuing to test this with an [interned string](https://www.phpinternalsbook.com/php7/internal_types/strings/zend_strings.html#interned-strings).